vegafusion-rt-datafusion/src/transform/utils.rs

```diff
@@ -9,22 +9,22 @@ use vegafusion_core::error::{Result, ResultWithContext, VegaFusionError};
 
 #[async_trait]
 pub trait DataFrameUtils {
-    fn block_eval(&self) -> Result<Vec<RecordBatch>>;
-    fn block_flat_eval(&self) -> Result<RecordBatch>;
-    async fn collect_flat(&self) -> Result<RecordBatch>;
+    fn block_eval(self) -> Result<Vec<RecordBatch>>;
+    fn block_flat_eval(self) -> Result<RecordBatch>;
+    async fn collect_flat(self) -> Result<RecordBatch>;
```

Comment:
DataFusion 16 made the DataFrame's methods consuming, so following that pattern here now